src/frontend/components/SubscribeScheduledVideoEmailForm/index.tsx

```diff
@@ -67,7 +68,8 @@ export const SubscribeScheduledVideoEmailForm = ({
   const intl = useIntl();
   const subscribeEmail = async () => {
     try {
-      await createLiveRegistration(email!);
+      const anonymousId = uuidv4();
```

Comment:
This is temporary ?

Reply:
yes, waiting for the data to exist in front